constants/index.js

```diff
@@ -1,2 +1,40 @@
 export const HOLES = [1, 2, 3, 4, 5, 6, 7, 8, 9];
 export const PAR = 27;
+
+export const PLAYERS = [
+  {
+    id: 'eric',
+    spreadsheetKey: 'eric g',
+    name: 'Eric Giovanola',
+  },
+  {
+    id: 'zamore',
```

Comment:
Whoa, "zamore", "fzamore", and "ericzamore" all redirect to the profile page! I didn't realize you could do that...